cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -91,7 +91,7 @@ def catalog_item(ansible_repository):
         fauxfactory.gen_alphanumeric(),
         fauxfactory.gen_alphanumeric(),
         provisioning={
-            "repository": ansible_repository,
+            "repository": ansible_repository.name,
```

Comment:
fillable+as_fill_value isn't providing this?

Reply:
It seems no